QuestionFive/QuestionFive/Program.cs

```diff
@@ -6,10 +6,107 @@
 
 namespace QuestionFive
 {
-    class Program
+    public class Program
     {
         static void Main(string[] args)
         {
+            int a = 0;
+
+            Values userInput = UserInput(a);
+            int h = userInput.x;
+            string verbFeed = EngRet(h);
+            Console.WriteLine();
+            Console.WriteLine("Your number in written form is '{0}'.", verbFeed);
+            Console.ReadLine();
+        }
+        
+        public static Values UserInput(int a)
```

Comment:
Because this routine's purpose is to return a single integer, it is not necessary for the integer to be contained inside a class (viz. `Values`).  However, it is not considered "wrong" to do what you did.